casper/src/main/scala/coop/rchain/casper/util/rholang/costacc/PreChargeDeploy.scala

```diff
@@ -35,27 +34,25 @@ object PreChargeDeploy extends SystemDeploy {
     )
 
   override val source: String =
-    """
-                                |new rl(`rho:registry:lookup`),
-                                |poSCh,
-                                |initialDeployerId(`sys:casper:initialDeployerId`),
-                                |chargeAmount(`sys:casper:chargeAmount`),
-                                |return(`sys:casper:return`) in
-                                |{
-                                |  rl!(`rho:rchain:pos`, *poSCh) |
-                                |  for(@(_, PoS) <- poSCh) {
-                                |    @PoS!("chargeDeploy", *initialDeployerId, *chargeAmount, *return)
-                                |  }
-                                |
-                                |}""".stripMargin
+    """|new rl(`rho:registry:lookup`),
```

Comment:
The same from my previous comment about alignment applies here as well.

Reply:
Also, fixed in 31424de3